### DISCUSSION TITLE
Command /cmi voteedit

### DISCUSSION THREAD

**Discussion by SnonSS** [Zrips/CMI#7671] [Enhancement]
Make it possible to set negative numbers with the / cmi voteedit set (nick) -1 command

**Comment by CloudeLecaw** [comment]
why the number of a vote players should have should be lower then 0¿

**Comment by SnonSS** [comment]
well why not

**Comment by CloudeLecaw** [comment]
at least you created this thread in the wrong area
use discussionfor requests https://github.com/Zrips/CMI/discussions
in 99% of all cases requests in issues will be ignored.
